proxy-recorder/launch-gatling-proxy.sh

```diff
@@ -169,6 +169,27 @@ read
 
 FIND_COMMAND='find ./user-files/bodies -name *.txt'
 FIND_COUNT=`${FIND_COMMAND} |wc -l`
+  
+# The find count will return the number of things found, which will
+# be greater than 1 when there are "leftover" results in the directories
+# due to past aborts (like with ^C).  Issue a warning about needing
+# a little house cleaning, then try to recover and continue on 
+#
+if [ ${FIND_COUNT} -gt 1 ]	# Use numeric comparison, not string
+then
+   echo "
+WARNING!
+There are left-over recording bodies in the \"./user-files/bodies\" directory 
+and the \"../simulation-runner/src/main/scala/com/puppetlabs/gatling/node_simulations/\"
+directory (.txt and .scala files).  If the recording information for them is complete, 
+then you can manually copy the existing files to the proper directories or delete them 
+and re-run this recording if necessary.
+
+Attempting to continue.
+"
+  FIND_COUNT=1			# Force completion for the current recording
```

Comment:
TBH, it's not clear to me that this would be the best behavior.  I'd actually prefer to see it bail out in this case like it is today since otherwise you'd run the risk of ending up with a polluted recording - or the recording process might fail downstream for a reason that might be hard to decipher.

One alternative to this might be to add a `--force` command line argument to the script which, when set, would nuke the content of the `./user-files/bodies` directory before the proxy recorder is run.  That way, the user would be in charge of deciding whether or not it is okay to delete the leftover artifacts from a prior run.

@rlinehan or @jpinsonault - do you have any specific thoughts on this one?